smartsim/entity/entity.py

```diff
@@ -107,17 +107,16 @@ def __init__(self, name: str) -> None:
         :param name: Name of the entity
         """
         self.name = name
+        """The name of the application"""
+
+    @abc.abstractmethod
+    def as_program_arguments(self) -> t.Sequence[str]: ...
```

Comment:
Can you add a quick docstring here telling the implementer what this method is supposed to do?